src/components/BooksList.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import Book from './Book';
+
+const BooksList = ({ books }) => (
+  <table>
+    <tr>
+      <th>Book ID</th>
+      <th>Title</th>
+      <th>Category</th>
+    </tr>
+    {books.map(book => <Book key={book.id} book={book} />)}
+  </table>
+);
+const mapStateToProps = state => ({
+  books: state.books,
+});
```

Comment:
- [ ] As it is requested in the instructions, we need to use the Presentational and Container Components approach. This means that components are divided based on their awareness of Redux. Container components are the ones that are aware of Redux existence and Presentation components are the ones that don't. Check the documentation [here](https://redux.js.org/basics/usage-with-react#presentational-and-container-components). Both BookList and BookForm (BookForm it's not connected to Redux yet but it's going to in a future milestone) should be placed in the `containers` directory.